source/templates/actions.md

```diff
@@ -117,7 +117,7 @@ When an action is triggered, but no matching action handler is
 implemented on the controller, the current route, or any of the
 current route's ancestors, an error will be thrown.
 
-![Action Bubbling](/images/template-guide/action-bubbling.png)
+<img src="/images/template-guide/action-bubbling.png" alt="Action Bubbling">
```

Comment:
This needs to be relative or it will break when published image lives at `v1.12.0/images/template-guide/action-bubbling.png` and not `/images/template-guide/action-bubbling.png`